### PR TITLE
Always initialise i18n

### PR DESCRIPTION
**Pull request by tephenavies**
### Checklist

-   [ ] There are unit tests to verify my changes are correct or unit tests aren't applicable (if so, write quick reason why unit tests don't exist)
-   [ ] I've updated CHANGES.md with what I changed.

### CODE REVIEW

**Review by nf-s — APPROVED** [review]
Hey @steve9164 I made a few changes;

- tsified `TerriaError` and `raiseErrorToUser`
- created `wrapErrorMessage` function to wrap error message in something more user friendly (using `models.raiseError.errorMessage` translation string)
- created a `TerriaError` if loading config fails

Are you happy with these changes?

File: lib/Core/TerriaError.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  sender?: unknown;
+  /** True if the user has seen this error; otherwise, false. */
+  raisedToUser?: boolean;
+}
+
+/**
+ * Represents an error that occurred in a TerriaJS module, especially an asynchronous one that cannot be raised
+ * by throwing an exception because no one would be able to catch it.
+ */
+export default class TerriaError implements TerriaErrorOptions {
```
Comment (tephenavies) [inline lib/Core/TerriaError.ts]: It seems weird to me that a class implements its own initialisation options. Is there anywhere that this property is used?

Comment (nf-s) [reply lib/Core/TerriaError.ts]: Yeah I just did that to drag across JSDoc comments. I can get rid of it

Comment (nf-s) [reply lib/Core/TerriaError.ts]: i got rid of it

**Pull request merged**